docarray/document/mixins/pydantic.py

```diff
@@ -72,7 +74,7 @@ def from_pydantic_model(cls: Type['T'], model: 'BaseModel') -> 'T':
                 # second time is at `from_dict/from_json`, it is unnecessary yet inevitable, the result string get
                 # converted into a binary string and encoded again.
                 # consequently, we need to decode two times here!
```

Comment:
The code comments above are still reasonable?

Reply:
removed